### ISSUE TITLE
Analysing how SAN claims have changed since ios14.5 release

### ISSUE THREAD

**Issue by georgekoridze** [georgekoridze/adjust#1]
Hello,

ios14.5 was released on the 26th of April. It'll be very helpful to know what type of behavior should we expect from each SAN (in terms of if they continue claiming consented/non-consented users' traffic).

Could we take last few dates and pull the following KPIs for each SAN individually:

For each of their claim: activity (install, session, etc.), OS version, Network, ATT Status, `tracking_enabled` flag that we include in the ping to SAN (last one might not be sent to all SAN and that param might have a different name for each SAN it's sent to)

Basically, we'd like to know which SAN are claiming traffic for consented users and for non-consented (some might claim both, some might claim only consented, some might not claim at all). This behavior could be different for ios13-, ios14 - ios14.4, and ios14.5+ users, that's why it's also interesting to see the OS version. Also SAN might behave differently for different inventories (e.g. Google might claim non-consented traffic for Search, but not for admob, etc.) so seeing Network name would let us check the behavior for each inventory.
